luarules/gadgets/unit_dragons_disguise.lua

```diff
@@ -21,9 +21,11 @@ local ValidUnitID = Spring.ValidUnitID
 local neutralUnits = {}
 local armourTurrets = {}
 for udid,ud in ipairs(UnitDefs) do
-    if string.find(ud.name, "cormaw") or string.find(ud.name, "armclaw")  then
-        armourTurrets[udid] = true
-    end
+	if ud.customParams then
```

Comment:
`customParams` is universally injected during postprocessing the defs, and will never be nil, so this check can be removed.